resources/public/api/conf/5.0/schemas/foreign_property_cumulative_summary_retrieve/def1/response.json

```diff
@@ -0,0 +1,198 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "For TY 2025-26 and after",
+  "description": "This endpoint enables you to retrieve the cumulative period income and expenses for a foreign property business. A National Insurance number, business ID, and tax year are required. Please note that this endpoint is only available for tax years starting from 2025-26.",
+  "type": "object",
+  "properties": {
+    "submittedOn": {
+      "type": "string",
+      "description": "The date the foreign property were submitted, in the format YYYY-MM-DDThh:mm:ss.SSSZ",
+      "example": "2025-07-07T10:59:47.544Z"
+    },
+    "fromDate": {
+      "type": "string",
+      "description": "The first day that the income and expenses period summary covers. Must conform to the format YYYY-MM-DD.",
+      "example": "2025-04-06"
+    },
+    "toDate": {
+      "type": "string",
+      "description": "The last day that the income and expenses period summary covers. Must conform to the format YYYY-MM-DD.",
+      "example": "2025-07-05"
+    },
+    "foreignProperty": {
+      "type": "array",
+      "description": "Array holding the income and expenses of the user's foreign property business for the period.",
+      "items": {
```

Comment:
Should we include `title` under items to indicate what the array of objects represents, as specified in the spec? i.e. `"title": "foreignPropertyEntry"` to render it as shown below?

<img width="795" alt="foreignPropertyEntry" src="https://github.com/user-attachments/assets/0f059fd4-5f4f-4094-a802-381dc69fce78">

Not sure if this what we should be doing or not 🤷‍♂️